### PR TITLE
create the hover

### PR DESCRIPTION
**Pull request by cristianoprogramador**
hover created

### CODE REVIEW

File: src/components/userCard.tsx [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
   const { data } = props;
   const imageUrl = data.imageUrl ?? "";
 
+  const handleMouseEnter = () => {
+    setShowModal(true);
+  };
+
+  const handleMouseLeave = () => {
+    setShowModal(false);
+  };
```
Comment (RenatoTMLima) [inline src/components/userCard.tsx]: Suggestion: Move this logic to a separate component that handles everything about tooltips.

File: src/components/userCard.tsx [hunk 2]
```diff
@@ -18,10 +28,30 @@ export function UserCard(props: UserCardProps) {
...
+          } transition-opacity duration-200 ease-in-out text-center min-w-max`}
+        >
+          <div className="border-b">Projetos Ativos</div>
+          {data.projects?.map((project, index) => (
+            <div key={index} className="b">
+              <div>{project}</div>
+            </div>
+          ))}
+        </div>
+      )}
```
Comment (RenatoTMLima) [inline src/components/userCard.tsx]: Suggestion: Create a separate component for tooltips, where the children should be the component you are binding the tooltip and property to accept ReactNode, in which case you can pass the content of the tooltip.

**Review by RenatoTMLima — CHANGES_REQUESTED** [review]

File: src/components/userCard.tsx (hunk 1 above)
Comment (cristianoprogramador) [reply src/components/userCard.tsx]: i dont think we need that

File: src/components/userCard.tsx (hunk 2 above)
Comment (cristianoprogramador) [reply src/components/userCard.tsx]: i created a new component tooltip just for that

File: src/components/avatar.tsx [hunk 3]
```diff
@@ -12,7 +12,7 @@ export function Avatar(props: AvatarProps) {
     <img
       src={imageUrl}
       alt="userPhoto"
-      className={twJoin("avatar", className)}
+      className={twJoin("rounded-full border-white border-2", className)}
```
Comment (RenatoTMLima) [inline src/components/avatar.tsx]: Could you please make the implementation of the classes outside of the className property?

**Review by RenatoTMLima — CHANGES_REQUESTED** [review]

File: src/components/avatar.tsx (hunk 3 above)
Comment (cristianoprogramador) [reply src/components/avatar.tsx]: we did that on https://github.com/cristianoprogramador/FE_SH/pull/12

**Review by RenatoTMLima — DISMISSED** [review]

**Review by RenatoTMLima — APPROVED** [review]

**Pull request merged**